cpp/velox/memory/VeloxMemoryManager.cc

```diff
@@ -378,7 +378,7 @@ VeloxMemoryManager::~VeloxMemoryManager() {
                << "ms as there are still outstanding memory resources. ";
   }
 #ifdef ENABLE_JEMALLOC
-  je_gluten_malloc_stats_print(NULL, NULL, NULL);
+  malloc_stats_print(NULL, NULL, NULL);
 #endif
```

Comment:
Does this feature actually have users? I see this is the only usage of `jemalloc.h` in Gluten.

Reply:
@zhztheplayer, just has internal users for debugging purpose. Yes, it's just used here currently.